modules/invibesBidAdapter.js

```diff
@@ -87,10 +109,25 @@ function buildRequest(bidRequests, bidderRequest) {
   const _placementIds = [];
   const _adUnitCodes = [];
   let _customEndpoint, _userId, _domainId;
-  let _ivAuctionStart = bidderRequest.auctionStart || Date.now();
+  let _ivAuctionStart = Date.now();
+  if (window.placementIds == undefined) {
+    window.placementIds = [];
+  }
+
+  if (isInfiniteScrollPage == false) {
+    updateInfiniteScrollFlag();
+  }
 
   bidRequests.forEach(function (bidRequest) {
     bidRequest.startTime = new Date().getTime();
+
+    if (window.placementIds.includes(bidRequest.params.placementId)) {
+      isPlacementRefresh = true;
+    }
+
+    window.placementIds.push(bidRequest.params.placementId);
```

Comment:
Why is this data also being set to the window? What other code is reading it?

Reply:
It's used for the case when the script is called multiple times, as the placementIds needs to be persistent.

Reply:
Can you provide more details in your PR description as to the context of these changes, and provide the use cases your adapter is covering for publishers. Thank you